browser/app/js/actions.js

```diff
@@ -402,6 +404,24 @@ export const setLoginError = () => {
   }
 }
 
+export const downloadAllasZip = (url, req, xhr) => {
+  return (dispatch) => {
+    xhr.open('PUT', url, true)
```

Comment:
in that case we should use POST instead of PUT,  PUT is not expected to do what is being done here.